src/globals/scss/govuk-frontend.scss

```diff
@@ -1,3 +1,11 @@
+// sass-lint:disable no-css-comments
+// start:devonly
```

Comment:
Is there a better way to do this devonly thing?

Can we use a variable that is set to default off?

Reply:
I've seen https://gist.github.com/Leimi/a9f125406c1efd916448

Reply:
or https://www.npmjs.com/package/gulp-sass-variables

Reply:
Not a big deal otherwise

Reply:
I think the boring option is separate entry points

Reply:
I'll raise an issue to think about this later, since we need this in ASAP for research.

Reply:
https://github.com/alphagov/govuk-frontend/issues/556